config/muxcable.py

```diff
@@ -257,3 +264,208 @@ def disable(port, target):
         sys.exit(CONFIG_FAIL)
     click.echo("loopback disable sucessful")
     sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def hwmode():
+    """start configuring cable in hw mode on a port"""
+    pass
+
+
+@hwmode.command()
+@click.argument('state', metavar='<operation_status>', required=True, type=click.Choice(["active", "standby"]))
+@click.argument('port', metavar='<port_name>', required=True, default=None)
+def state(state, port):
+    """Configure the muxcable to {active/standby} hwmode mode"""
```

Comment:
Suggest rewording: `Configure the muxcable mux state` or similar

Reply:
fixed